src/routes/committee/schedule/[rehearsalId]/+page.ts

```diff
@@ -49,19 +49,22 @@ export const load: PageLoad = async ({ params }) => {
 		songs = songDocs.map((doc) => ({ id: doc.id, ...doc.data() } as song))
 
 		// Get the playsSongInEdition for all participants on this rehearsal day
-		const playsInQuery = query(
-			collectionGroup(db, 'playsSongInEdition'),
-			where(
-				'song',
-				'in',
-				songDocs.map((song) => song.ref)
-			)
-		)
+		// const playsInQuery = query(
+		// 	collectionGroup(db, 'playsSongInEdition'),
+		// 	where(
+		// 		'song',
+		// 		'in',
+		// 		songDocs.map((song) => song.ref)
+		// 	)
+		// )
+		// const playsInDocs = (await getDocs(playsInQuery)).docs
+
+		const playsInQuery = query(collectionGroup(db, 'playsSongInEdition'))
 		const playsInDocs = (await getDocs(playsInQuery)).docs
 
 		// Get all the participants that play these songs
 		const participantIds = playsInDocs.map((doc) => doc.ref.parent.parent?.id as string)
-		const participantQuery = query(collection(db, 'users'), where('__name__', 'in', participantIds))
+		const participantQuery = query(collection(db, 'users')) //, where('__name__', 'in', participantIds))
```

Comment:
Why remove the where filter? (also below and above)

Reply:
Because of the Firestore limit of max 10 in an 'in' filter. We need to write a helper function that correctly pulls the data, but for a quick fix this works.

Reply:
Understood 👍